### PR TITLE
show schedule by month on admin page

### PR DESCRIPTION
**Pull request by namframgia**

### CODE REVIEW

File: app/Repositories/WorkingScheduleRepository.php [hunk 1]
```diff
@@ -11,11 +13,62 @@ public function model()
...
+            if ($request->has('program') && $request->program !== null) {
+                $query->where('program_id', $request->program);
+            }
+            if ($request->has('position') && $request->position !== null) {
+                $query->where('position_id', $request->position);
+            }
+        })
+        ->get();
+
+        $first_day = Date(Date('Y') . '-' . Date('m') . '-' . '01');
```
Comment (framgiaciv3) [inline app/Repositories/WorkingScheduleRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/WorkingScheduleRepository.php</br>**Line**: 50</br>Variable "first_day" is not in valid camel caps format

File: app/Repositories/WorkingScheduleRepository.php [hunk 2]
```diff
@@ -11,11 +13,62 @@ public function model()
...
+        $last_day = Date(Date('Y') . '-' . Date('m') . '-' . Date('t'));
```
Comment (framgiaciv3) [inline app/Repositories/WorkingScheduleRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/WorkingScheduleRepository.php</br>**Line**: 51</br>Variable "last_day" is not in valid camel caps format

File: app/Repositories/WorkingScheduleRepository.php [hunk 3]
```diff
@@ -11,11 +13,62 @@ public function model()
...
+
+        $schedules = $this->model->whereBetween('date', [$first_day, $last_day])->get();
```
Comment (framgiaciv3) [inline app/Repositories/WorkingScheduleRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/WorkingScheduleRepository.php</br>**Line**: 53</br>Variable "first_day" is not in valid camel caps format

Comment (framgiaciv3) [inline app/Repositories/WorkingScheduleRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/WorkingScheduleRepository.php</br>**Line**: 53</br>Variable "last_day" is not in valid camel caps format

File: app/Repositories/WorkingScheduleRepository.php [hunk 4]
```diff
@@ -11,11 +13,62 @@ public function model()
...
+        $data = collect();
+        foreach ($schedules as $schedule) {
+            $day = \DateTime::createFromFormat("d-m-Y", $schedule->date)->format('j');
```
Comment (framgiaciv3) [inline app/Repositories/WorkingScheduleRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Repositories/WorkingScheduleRepository.php</br>**Line**: 56</br>String "d-m-Y" does not require double quotes; use single quotes instead

**Comment by namframgia** [comment]
https://edu-redmine.framgia.vn/issues/5610

**Pull request merged**